### DISCUSSION TITLE
developer-based income

### DISCUSSION THREAD

**Discussion by DaviRain-Su** [creatorsand/.github#53] [Ideas]
一个使得开发者，或者开发初学者可以来参与贡献代码的平台，并且综合开发的忠诚度，或者各种评分，根据开发者贡献的代码，评分开发者的技能水平。

1. 是什么,为开发服务的一个问题聚合信息网站，开发者可以通过这个网站查询你自己感兴趣的问题，解决贡献你的代码。
2. 为什么，因为在区块世界中，有良好的空投传统，celestia项目开创了一个开始，讲token分发给项目的贡献者，这也是需要这个平台的原因
3. 怎么做，需要收集不错的还未release token的项目，或者一些项目与我们合作发布issue或者问题信息
4. 需要什么角色，运营，开发，用户，问题领队老师，项目方或者优质项目收集者
5. 什么时候上线 ， 能越快上线越好了啊

**Comment by ShirleneLiu** [comment]
我可以来协助 bala 做设计（仅此了
